code-build.js

```diff
@@ -222,6 +245,12 @@ function githubInputs() {
   const disableGithubEnvVars =
     core.getInput("disable-github-env-vars", { required: false }) === "true";
 
+  const stopOnSignals = core
+    .getInput("stop-on-signals", { required: false })
```

Comment:
do we need to set the default value `SIGINT` here?

Reply:
got it.  I asked this question because of the issue I reported in the other comment.